k8s/upgrade/src/upgrade_resources/upgrade.rs

```diff
@@ -5,12 +5,31 @@ use crate::{
         UPGRADE_JOB_IMAGE, UPGRADE_JOB_NAME_SUFFIX, UPGRADE_JOB_SERVICEACCOUNT_NAME_SUFFIX,
     },
     upgrade_resources::objects,
+    user_prompt,
 };
+
+#[derive(clap::Subcommand, Debug)]
+/// Actions to be performed.
+pub enum PodLabels {
+    IoEnginePodLabel,
+    AgentCorePodLabel,
+    ApiRestPodLabel,
+}
+
+impl PodLabels {
+    fn value(&self) -> &str {
+        match *self {
+            PodLabels::IoEnginePodLabel => "app=io-engine",
```

Comment:
Should these come from the constants module?
Even better perhaps would be to read these directly from helm :)